projects/ngrx.io/content/guide/store/actions.md

```diff
@@ -55,52 +53,58 @@ Following these guidelines helps you follow how these actions flow throughout yo
 Let's look at an example action of initiating a login request.
 
 <code-example header="login-page.actions.ts">
-import { Action } from '@ngrx/store';
-
-export class Login implements Action {
-  readonly type = '[Login Page] Login';
+import { createAction, props } from '@ngrx/store';
 
-  constructor(public payload: { username: string; password: string }) {}
-}
+export const login = createAction(
+  '[Login Page] Login',
+  props<{ username: string; password: string }>()
+);
 </code-example>
 
-Actions are written as classes to provide a type-safe way to construct an action when it's being dispatched. The `Login` action implements the `Action` interface to adhere to its structure. The `payload` in this example is an object of a username and password, that is additional metadata needed for the handling of the action.
+The `createAction` function returns a function, that when called returns an object in the shape of the `Action` interface. The `props` method is used to define any additional metadata needed for the handling of the action. Actions creators provide a consistent, type-safe way to construct an action that is being dispatched.
```

Comment:
Action creators provide a consistent, type-safe way to construct an action that is being dispatched.